static/js/stream_ui_updates.js

```diff
@@ -172,7 +172,7 @@ exports.update_subscribers_list = function (sub) {
     if (!sub.can_access_subscribers) {
         $(".subscriber_list_settings_container").hide();
     } else {
-        const emails = stream_edit.get_email_of_subscribers(sub.subscribers);
+        const users = stream_edit.get_person_from_subscribers(sub.subscribers);
```

Comment:
I renamed this to `get_users_from_subscribers`, since that makes more clear it's returning several users.